packages/kit-bg/src/vaults/impls/kaspa/Vault.ts

```
@@ -204,8 +241,31 @@
         to: utxoTo[0].address,
       },
     };
-
-    if (swapInfo) {
+    if (
+      transferPayload &&
+      transferPayload.tokenInfo &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)